src/Microsoft.Azure.SignalR.Management/IInternalServiceHubContext.cs

```diff
@@ -11,29 +11,20 @@
 
 namespace Microsoft.Azure.SignalR.Management
 {
-    public interface IServiceContext : IDisposable
+    internal interface IInternalServiceHubContext : IServiceHubContext
     {
         /// <summary>
-        /// Creates an instance of <see cref="IServiceHubContext"/> asynchronously.
+        /// Gets client endpoint access information object for SignalR hub connections to connect to Azure SignalR Service
         /// </summary>
-        /// <param name="hubName">The hub name.</param>
-        /// <param name="cancellationToken">Cancellation token for creating service hub context.</param>
-        /// <returns>The task object representing the asynchronous operation.</returns>
-        Task<IServiceHubContext> CreateHubContextAsync(string hubName, CancellationToken cancellationToken = default);
-
-        /// <summary>
-        /// Gets <see cref="NegotiationResponse"/> for SignalR hub connections to connect to Azure SignalR Service
-        /// </summary>
-        /// <param name="hubName">The hub name.</param>
         /// <param name="httpContext">The HTTP context which might provide information for routing and generating access token.</param>
         /// <param name="userId">The user ID. If null, the identity name in <see cref="HttpContext.User" /> of <paramref name="httpContext"/> will be used.</param>
         /// <param name="claims">The claim list to be put into access token. If null, the claims in <see cref="HttpContext.User"/> of <paramref name="httpContext"/> will be used.</param>
         /// <param name="lifetime">The lifetime of the token. The default value is one hour.</param>
         /// <param name="isDiagnosticClient">The flag whether the client to be connected is a diagnostic client.</param>
         /// <param name="cancellationToken">Cancellation token for aborting the operation. If null, the <see cref="HttpContext.RequestAborted"/> of <paramref name="httpContext"/> will be used. </param>
         /// <returns>Client endpoint and access token to Azure SignalR Service.</returns>
-        Task<NegotiationResponse> NegotiateAsync(string hubName, HttpContext httpContext = null, string userId = null, IList<Claim> claims = null, TimeSpan? lifetime = null, bool isDiagnosticClient = false, CancellationToken cancellationToken = default);
+        Task<NegotiationResponse> NegotiateAsync(HttpContext httpContext = null, string userId = null, IList<Claim> claims = null, TimeSpan? lifetime = null,bool isDiagnosticClient = false, CancellationToken cancellationToken = default);
```

Comment:
`null,bool` space

Reply:
Emmm, actually it is a newly created file, but git associates it with an old deleted file. The new space is added by formatter. I think it's better to keep it, otherwise we have to deal with the space every time we format the file.